bin/verify-exercises

```diff
@@ -1,40 +1,79 @@
 #!/usr/bin/env bash
 
 # Synopsis:
-# Test the track's exercises.
-# 
-# At a minimum, this file must check if the example/exemplar solution of each 
-# Practice/Concept Exercise passes the exercise's tests.
-#
-# To check this, you usually have to (temporarily) replace the exercise's solution files
-# with its exemplar/example files.
-#
-# If your track uses skipped tests, make sure to (temporarily) enable these tests
-# before running the tests.
-#
-# The path to the solution/example/exemplar files can be found in the exercise's
-# .meta/config.json file, or possibly inferred from the exercise's directory name.
+# Verify that each exercise's example/exemplar solution passes the tests.
+# You can either verify all exercises or a single exercise.
 
 # Example: verify all exercises
-# ./bin/verify-exercises
+# bin/verify-exercises
 
 # Example: verify single exercise
-# ./bin/verify-exercises two-fer
+# bin/verify-exercises two-fer
 
-slug="${1:-*}"
+set -eo pipefail
 
-# Verify the Concept Exercises
-for concept_exercise_dir in ./exercises/concept/${slug}/; do
-    if [ -d $concept_exercise_dir ]; then
-        echo "Checking $(basename "${concept_exercise_dir}") exercise..."
-        # TODO: run command to verify that the exemplar solution passes the tests
-    fi
-done
+required_tool() {
+    command -v "${1}" >/dev/null 2>&1 ||
+        die "${1} is required but not installed. Please install it and make sure it's in your PATH."
+}
+
+required_tool jq
+
+copy_example_or_examplar_to_solution() {
+    jq -c '[.files.solution, .files.exemplar // .files.example] | transpose | map({src: .[1], dst: .[0]}) | .[]' .meta/config.json | while read -r src_and_dst; do
```

Comment:
```suggestion
    jq -c '[.files.solution, .files.exemplar // .files.example] | transpose | map({src: .[1], dst: .[0]}) | .[]' .meta/config.json \
    | while read -r src_and_dst; do
```
Just adding a newline for readability.

Are there exercises where the number of solution files != number of example files?

Reply:
No, they should always match